css/styles.css

```diff
@@ -22,10 +22,64 @@ header .nav a {
   color: #28352f;
 }
 
-.header ul {
+.menu-desktop {
   display: none;
 }
 
+.menu-bar {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  top: 0%;
+  right: 0%;
+  width: 100%;
+  height: 820px;
+  z-index: 10;
+  background-color: white;
```

Comment:
![screenshot (9)](https://user-images.githubusercontent.com/44624138/145990442-6744faa8-19c8-4d63-bc72-5638d6d3073b.png)

- [ ] I'd suggest you use `height: 100%` and the `position: fixed` so the menu fills the screen and it does not scroll with the page🤔

Reply:
Ok. Thank you. Fixed.